03_launch_mgmt_cluster.sh

```diff
@@ -10,6 +10,7 @@ source lib/releases.sh
 # shellcheck disable=SC1091
 source lib/network.sh
 
+export CAPI_CONFIG_FOLDER="${HOME}/.cluster-api"
```

Comment:
according to this condition https://github.com/metal3-io/metal3-dev-env/pull/1474/files#diff-417b79700310fc30a2ae54bd699206962d6a97cfe30018a66ba6854312acdf9dL114 
CAPI_CONFIG_FOLDER should be set like this:
```
  # Default CAPI_CONFIG_FOLDER to $HOME/.config folder if XDG_CONFIG_HOME not set
  CONFIG_FOLDER="${XDG_CONFIG_HOME:-$HOME/.config}"
  export CAPI_CONFIG_FOLDER="${CONFIG_FOLDER}/cluster-api"
```